flow/designs/asap7/mock-array-big/macro-placement.tcl

```diff
@@ -1,18 +1,38 @@
 # fish out values from single source of truth: config.py
-set vals [regexp -all -inline {\S+} [exec sh -c {cd designs/asap7/mock-array-big && python3 -c "import config;print(f'{config.rows} {config.cols} {config.pitch_and_margin} {config.die_offset_x} {config.die_offset_y}')"}]]
-lassign $vals rows cols pitch_and_margin die_offset_x die_offset_y
+set vals [regexp -all -inline {\S+} [exec sh -c {cd designs/asap7/mock-array-big && python3 -c "import config;print(f'{config.rows} {config.cols} {config.ce_pitch_x} {config.ce_pitch_y} {config.margin_x} {config.margin_y} {config.placement_grid_x} {config.placement_grid_y}')"}]]
+lassign $vals rows cols ce_pitch_x ce_pitch_y margin_x margin_y placement_grid_x placement_grid_y
 
 set block [ord::get_db_block]
 set units [$block getDefUnits]
 
+set x [expr ($margin_x + ($ce_pitch_x / 2)) * $units]
+set y [expr ($margin_y + ($ce_pitch_y / 2)) * $units]
+
+set instList [list]
+for {set i 0} {$i < 8} {incr i} {
```

Comment:
This code should be reverted, such that other configurations like 1x1, 16x8 can be supported.

I can't surmise why it was changed to be specific to 8x8 configurations.